spread.yaml

```diff
@@ -52,9 +52,13 @@ prepare: |
   mv chisel /usr/local/bin
 
 suites:
-  tests/:
-    summary: Tests common scenarios
+  tests/common/:
```

Comment:
What is common and what is specific?  This feels extremely subjective, which means we'll have no idea about what tests are supposed to go on which, both when writing them and when trying to find them.